esp/esp/survey/migrations/0002_auto_20200301_1048.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def create_question_types(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    QuestionType = apps.get_model("survey", "QuestionType")
+    QuestionType.objects.using(db_alias).get_or_create(name="Instruction Text", _param_names = "Text")
+    QuestionType.objects.using(db_alias).get_or_create(name="Long Answer", _param_names = "Rows|Columns")
```

Comment:
I sorta understand that rows will make the input box longer, but can there just be a default for the columns business? It feels really unnatural to specify the columns for a long answer question.

Reply:
Actually I'm not even sure changing the columns does anything at least in the rendered questions display in this builder.